### ISSUE TITLE
Getting `mkdir: : No such file or directory` on any nave 0.5.2 invocation

### ISSUE THREAD

**Issue by mhart** [isaacs/nave#81]
(on Mac OS X 10.11, Node v4.1.1)

Here's the debug output:

```console
$ NAVE_DEBUG=true nave
+ '[' -z /bin/bash ']'
++ basename /usr/local/bin/bash
+ shell=bash
+ case "$shell" in
+ shopt -s extglob
++ uname -a
+ uname='Darwin ReBuke-Pro.local 15.0.0 Darwin Kernel Version 15.0.0: Wed Aug 26 16:57:32 PDT 2015; root:xnu-3247.1.106~1/RELEASE_X86_64 x86_64'
+ os=
+ arch=
+ case "$uname" in
+ os=darwin
+ case "$uname" in
+ arch=x64
+ tar=tar
+ main
+ local SELF_PATH DIR SYM
+ SELF_PATH=/usr/local/bin/nave
+ '[' / '!=' . ']'
+ '[' / '!=' / ']'
+++ dirname -- /usr/local/bin/nave
++ cd -P -- /usr/local/bin
++ pwd -P
+ SELF_PATH=/usr/local/bin
++ basename -- /usr/local/bin/nave
+ SELF_PATH=/usr/local/bin/nave
+ '[' -h /usr/local/bin/nave ']'
++ dirname -- /usr/local/bin/nave
+ DIR=/usr/local/bin
++ readlink -- /usr/local/bin/nave
+ SYM=../lib/node_modules/nave/nave.sh
++ cd -- /usr/local/bin
+++ dirname -- ../lib/node_modules/nave/nave.sh
++ cd -- ../lib/node_modules/nave
++ pwd
++ basename -- ../lib/node_modules/nave/nave.sh
+ SELF_PATH=/usr/local/lib/node_modules/nave/nave.sh
+ '[' -h /usr/local/lib/node_modules/nave/nave.sh ']'
+ '[' -z '' ']'
+ '[' -d '' ']'
+ mkdir -p -- ''
mkdir: : No such file or directory
```

**Comment by mhart** [comment]
Could be due to this recent change? https://github.com/isaacs/nave/commit/c28ce962a71de4666a585df59d10b7531e60ae44

**Comment by mhart** [comment]
This is under bash 3.2.57 if that helps:
```console
$ /bin/bash --version
GNU bash, version 3.2.57(1)-release (x86_64-apple-darwin15)
Copyright (C) 2007 Free Software Foundation, Inc.
```

**Comment by mhart** [comment]
Soooo... I'm guessing this was fixed in #83 ?

**Comment by mhart** [comment]
Pretty sure it was, gonna close this.
